src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -310,7 +310,7 @@ private static string GetUri(HttpRequest request)
             var queryString = request.QueryString.Value ?? string.Empty;
             var length = scheme.Length + Uri.SchemeDelimiter.Length + host.Length + pathBase.Length
                          + path.Length + queryString.Length;
-#if NETSTANDARD2_1
+#if !NETSTANDARD2_0
```

Comment:
Would it be possible to describe this in a way that reflects the actual intention?

e.g. `#if NET5_0_OR_GREATER || NETSTANDARD2_1_OR_GREATER`